utils/midiplay/opnplay.cpp

```diff
@@ -237,6 +237,12 @@ int main(int argc, char **argv)
 
     std::fprintf(stdout, " - %s Emulator in use\n", opn2_chipEmulatorName(myDevice));
 
+#if defined(OPNMIDI_ENABLE_HQ_RESAMPLER)
+    std::fprintf(stdout, " - Resampling by the high quality method\n");
+#else
+    std::fprintf(stdout, " - Resampling by the simple method\n");
+#endif
```

Comment:
I think, this probably can be implemented as API function which will statically return the integer that will mean HQ resampler or simple. Like `linkedVersion` call does. That will give more truth when we originally built this and dynamically linked with HQ, but then replaced SO with simple. In result - a false phrase :yum:

Reply:
Yeah you're right, but I don't like too much to set it in stone, because for me this it's experimental purpose like a/b testing, and practical usefulness of elaborate resampling must yet be shown.

While linear resampling vs better methods make night and day in some contexts, like in sample-based tracker software, I have not heard stellar differences out of ADLMIDI software in a few situations I have tried.